clients/da-vinci-client/src/test/java/com/linkedin/davinci/config/DaVinciConfigTest.java

```diff
@@ -45,25 +40,36 @@ public void processPut(Lazy<Integer> key, Lazy<Integer> value) {
   public void testRecordTransformerEnabled() {
     DaVinciConfig config = new DaVinciConfig();
     assertFalse(config.isRecordTransformerEnabled());
+
     DaVinciRecordTransformerConfig recordTransformerConfig = new DaVinciRecordTransformerConfig(
-        (storeVersion) -> new TestRecordTransformer(storeVersion, true),
-        Integer.class,
+        (_storeVersion, _keySchema, _originalValueSchema, _outputValueSchema) -> new TestRecordTransformer(
+            _storeVersion,
+            _keySchema,
+            _originalValueSchema,
+            _outputValueSchema,
```

Comment:
Nitpick: I would prefer that we don't use underscores since we don't use them in the rest of the code base. Moreover, in some coding styles (not in ours, however) the underscore at the beginning is used to denote a class property, which is not the case here, but could still confuse some readers...

There are several other occurrences of underscores in lambda params below. Can we please change all of them?